android/java/org/chromium/chrome/browser/preferences/BravePrefServiceBridge.java

```diff
@@ -0,0 +1,49 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.preferences;
+
+import org.chromium.base.ThreadUtils;
+
+public class BravePrefServiceBridge extends PrefServiceBridge {
```

Comment:
@yrliou I have question about this inheritance.
Why did you inherit `PrefServiceBridge` for `BravePrefServiceBridge`?
We are using two instances for `BravePrefServiceBridge` and `PrefServiceBridge`.
and I think `BravePrefServiceBridge` can work w/o `PrefServiceBridge` impl.